packages/lib/server/repository/booking.ts

```diff
@@ -57,14 +58,16 @@ export class BookingRepository {
     });
   }
 
-  static async getAllBookingsForRoundRobin({
+  static async getBookingsForRoundRobin({
     users,
     eventTypeId,
+    fetchFromPastDays = null,
```

Comment:
Not needed to assign

Reply:
it should not make a difference, we are initialising with null for optional params, any way updated.